docs/project/faq.md

```diff
@@ -310,6 +313,84 @@ actively encourage using these languages in those cases. However, we need a
 solution for C++ use-cases that require its full performance, low-level control,
 and access to hardware.
 
+## Why is Carbon designed that way?
+
+### Why are there no angle brackets?
+
+Angle brackets cause major parsing problems in a language that also uses `<` and
+`>` as comparison operators. In particular,
+[one of our sub-goals for Carbon](/docs/project/goals.md#fast-and-scalable-development)
+is that it should support parsing without contextual or semantic information,
+and experience with C++ has shown that angle brackets make that difficult if not
+impossible.
```

Comment:
> Also, keep an eye on explaining what the use _other_ than comparison operators is: again, that's not explained here.

As above, that's intentional; I don't want to focus on templates because I want to explain why we can't use angle brackets anywhere else either.

I think I've addressed the rest of this comment.